beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -224,6 +225,15 @@ impl<AppReqId: ReqId, TSpec: EthSpec> Network<AppReqId, TSpec> {
             // Set up a scoring update interval
             let update_gossipsub_scores = tokio::time::interval(params.decay_interval);
 
+            let max_subnets = ctx.chain_spec.attestation_subnet_count as usize
```

Comment:
This is more like `max_topics` rather than `max_subnets`

Reply:
Thanks, resolved